packages/docs/src/pages/en/page-exports/relay.md

```diff
@@ -73,14 +73,14 @@ export const Page = ({ queryRef }) => {
 
 While using Relay, it's pretty common to pass query variables for fetching data.
 Vilay provides `getQueryVariables` to let users return variables to use.
-It receives [route parameters](https://vite-plugin-ssr.com/filesystem-routing) which can be used to figure out the right query variables.
+It receives page context such as [route parameters](https://vite-plugin-ssr.com/filesystem-routing) which can be used to figure out the right query variables.
```

Comment:
Adding a link to VPS pageContext docs (https://vite-plugin-ssr.com/pageContext) would be nice.